01-node-tutorial/answers/12-http-HVD.js

```diff
@@ -0,0 +1,17 @@
+const http = require('http')
+
+const server = http.createServer((req, res) => {
+    if (req.url === '/'){
+        res.end('Welcome to our homepage.') 
+    }
+    else if (req.url === '/about'){
```

Comment:
```suggestion
    } else if (req.url === '/about') {
```

Another formatting nit, but this one is sightly more meaningful. It can be easier to scan a big chunk of `if`, `if else`, and `else` blocks if you bring the `else if` and `else` keywords onto the same line as the closing curly brace that precedes it.